Modules/Core/Common/include/itkImageBase.hxx

```diff
@@ -528,16 +528,18 @@ ImageBase<VImageDimension>::PrintSelf(std::ostream & os, Indent indent) const
 
   os << indent << "Origin: " << this->GetOrigin() << std::endl;
 
-  os << indent << "Direction: " << std::endl << this->GetDirection() << std::endl;
+
+  os << indent << "Direction: " << std::endl;
+  this->GetDirection().PrintSelf(os, indent);
```

Comment:
This should probably be `indent` -> `indent.GetNextIndent()`

Reply:
I think so.